x-pack/plugin/searchable-snapshots/src/main/java/org/elasticsearch/xpack/searchablesnapshots/cache/blob/BlobStoreCacheMaintenanceService.java

```diff
@@ -60,14 +96,80 @@
 
     private static final Logger logger = LogManager.getLogger(BlobStoreCacheMaintenanceService.class);
 
+    /**
+     * The interval at which the periodic cleanup of the blob store cache index is scheduled.
+     */
+    public static final Setting<TimeValue> SNAPSHOT_SNAPSHOT_CLEANUP_INTERVAL_SETTING = Setting.timeSetting(
```

Comment:
I have no idea where the documentation for these settings should go. `searchable-snapshots/index.asciidoc` is more of a high-level explanation of the feature. Maybe in the Mount API since this is what triggers the creation of the system index?

Reply:
I think we can put it in the high level description for now (also contains the cache size settings). Maybe @jrodewig can then figure out if a new settings page should be added as a follow-up?

Reply:
Thanks @tlrx @henningandersen.

I agree that adding this setting to high-level page is probably to kay for now. That page currently include docs for some of the cache-related settings, like `xpack.searchable.snapshot.shared_cache.size`.

I'll defer to @debadair on whether to create a separate page in the long term. She's the docs lead for things related to searchable snapshots and data tiers.

Reply:
Thanks both! I pushed 8909ab8c291abb85d036dccb8d0672898dae2af4, suggestions welcome